documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc

```diff
@@ -46,7 +47,7 @@ NOTE: Feature gates might be removed when they reach GA. This means that the fea
 ¦`UseStrimziPodSets`
 ¦0.28
 ¦0.30
-¦ -
+¦0.35 (plan)
```

Comment:
```suggestion
¦0.35 (planned)
```

Reply:
Damn, I had it there and decided to change it later :-/.